R/agyw-integration.R

```diff
@@ -88,65 +88,64 @@ agyw_format_naomi <- function(outputs, options){
 
   # Not all country names in meta_area match the country names in the spreadsheet - need
   # to match to populate tabs
-  # country <- outputs$meta_area$area_name[outputs$meta_area$area_id == options$area_scope]
-  country <- dplyr::mutate(data.frame(iso3 = options$area_scope),
-                           country = fct_recode(iso3,
-                                             "Botswana" = "BWA",
-                                             "Cameroon" = "CMR",
-                                             "Kenya" = "KEN",
-                                             "Lesotho" = "LSO",
-                                             "Mozambique" = "MOZ",
-                                             "Malawi" = "MWI",
-                                             "Namibia" = "NAM",
-                                             "Eswatini" = "SWZ",
-                                             "Tanzania" = "TZA",
-                                             "Uganda" = "UGA",
-                                             "South Africa" = "ZAF",
-                                             "Zambia" = "ZMB",
-                                             "Zimbabwe" = "ZWE",
-                                             "Angola" = "AGO",
-                                             "Burundi" = "BDI",
-                                             "Democratic Republic of the Congo" = "COD",
-                                             "Gabon" = "GAB",
-                                             "Rwanda" = "RWA",
-                                             "Ethiopia" = "ETH",
-                                             "Haiti" = "HTI",
-                                             "Chad" = "TCD",
-                                             "Cote D'Ivoire" = "CIV",
-                                             "Ghana" = "GHA",
-                                             "Guinea" = "GIN",
-                                             "Liberia" = "LBR",
-                                             "Mali" = "MLI",
-                                             "Niger" = "NER",
-                                             "Sierra Leone" = "SLE",
-                                             "Togo" = "TGO",
-                                             "Burkina Faso" = "BFA",
-                                             "Congo" = "COG",
-                                             "Benin" = "BEN",
-                                             "Central African Republic" = "CAF",
-                                             "The Gambia" = "GMB",
-                                             "Guinea bissau" = "GNB",
-                                             "Equatorial Guinea" = "GNQ",
-                                             "Niger" = "NER",
-                                             "Nigeria" = "NGA",
-                                             "Senegal" = "SEN"))
-
-
 
+  country_name_db <- tibble::tribble(~country, ~iso3,
```

Comment:
@krisher1 I have updated the method for hardcoding the country names to pulling straight from a dataframe.